ddl/db_integration_test.go

```diff
@@ -1727,6 +1727,18 @@ func (s *testIntegrationSuite3) TestAlterColumn(c *C) {
 	tk.MustExec("drop table multi_unique")
 	tk.MustExec("create table multi_unique (a serial serial default value serial default value)")
 	tk.MustExec("drop table multi_unique")
+
+	tk.MustExec("drop table if exists t1")
+	tk.MustExec("create table t1(id int)")
+	tk.MustExec("insert into t1(id) values (1);")
+	tk.MustExec("alter table t1 add column update_time datetime NOT NULL DEFAULT CURRENT_TIMESTAMP();")
+	tk.MustExec("alter table t1 add column update_time3 datetime(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3);")
+	tk.MustExec("alter table t1 add column update_time6 datetime(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6);")
+	rows := tk.MustQuery("select * from t1").Rows()
+	c.Assert(rows[0][0], Equals, "1")
+	c.Assert(len(rows[0][1].(string)), Equals, 19)
+	c.Assert(len(rows[0][2].(string)), Equals, 23)
```

Comment:
Before this  PR the result is `2021-08-13 13:45:12.000`, the length feels the same before and after PR, so what is the point of this test?

Reply:
ah.... you're right, this test tests nothing, try to fix.

Reply:
What about a fixed manual default value?